index.html

```diff
@@ -24,19 +24,200 @@
     <div class="menu-bar"><i class="fa-solid fa-bars"></i></div>
   </header>
   <section class="intro-container">
-    <h1 class="main-element">I’m Olivier <br/>Glad to see you!</h1>
+    <h1 class="main-element">I’m Olivier <br />Glad to see you!</h1>
     <p class="main-element">
       I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
     </p>
     <button type="button" class="main-element">LET’S CONNECT</button>
     <ul class="main-element">
-      <li><a href="https://twitter.com/i/flow/login?input_flow_data=%7B%22requested_variant%22%3A%22eyJsYW5nIjoiZW4ifQ%3D%3D%22%7D" target=”_blank” class="media-icon"><i class="fa-brands fa-twitter"></i></a></li>
-      <li><a href="https://www.linkedin.com/" target=”_blank” class="media-icon"><i class="fa-brands fa-linkedin-in"></i></a></li>
-      <li><a href="https://medium.com/" target=”_blank” class="media-icon"><i class="fa-solid fa-m"></i></a></li>
-      <li><a href="https://github.com/" target=”_blank” class="media-icon"><i class="fa-brands fa-github"></i></a></li>
-      <li><a href="https://angel.co/" target=”_blank” class="media-icon"><i class="fa-brands fa-angellist"></i></a></li>
+      <li>
+        <a href="https://twitter.com/i/flow/login?input_flow_data=%7B%22requested_variant%22%3A%22eyJsYW5nIjoiZW4ifQ%3D%3D%22%7D" target="”_blank”" class="media-icon"><i class="fa-brands fa-twitter"></i
+					></a>
+      </li>
+      <li>
+        <a href="https://www.linkedin.com/" target="”_blank”" class="media-icon"><i class="fa-brands fa-linkedin-in"></i
+					></a>
+      </li>
+      <li>
+        <a href="https://medium.com/" target="”_blank”" class="media-icon"><i class="fa-solid fa-m"></i
+					></a>
+      </li>
+      <li>
+        <a href="https://github.com/" target="”_blank”" class="media-icon"><i class="fa-brands fa-github"></i
+					></a>
+      </li>
+      <li>
+        <a href="https://angel.co/" target="”_blank”" class="media-icon"><i class="fa-brands fa-angellist"></i
+					></a>
+      </li>
     </ul>
   </section>
+  <section class="work-container">
+    <div class="work-wrapper">
+      <div class="work-sub-cont work-img-wrapper">
+        <img src="./images/tonic.jpg" alt="work pictictre" />
```

Comment:
- [ ] Make sure that you follow correctly Figma design for your first work card.
### Figma design: 
![image](https://user-images.githubusercontent.com/71874425/159026868-b862a47f-415f-4a23-942a-a0ad88c8f22b.png)
### Your design: 
![image](https://user-images.githubusercontent.com/71874425/159027434-a08f3e9d-a829-4e25-9aff-1e572cf63e7e.png)
- Please, kindly fix this issue by fixing the image above your content.